src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ContestedOrderController.java

```diff
@@ -64,31 +62,22 @@
 
         CaseDetails caseDetails = callback.getCaseDetails();
         Map<String, Object> caseData = caseDetails.getData();
-        List<String> errors = new ArrayList<>();
 
-        List<String> contestedCaseOrderErrors = contestedCaseOrderService.printAndMailGeneralOrderToParties(caseDetails, authToken);
-        errors.addAll(contestedCaseOrderErrors);
+        contestedCaseOrderService.printAndMailGeneralOrderToParties(caseDetails, authToken);
 
-        if (!isPaperApplication(caseData)) {
```

Comment:
you've removed the paper application check, is that intentional?

Reply:
yes, it was not needed - no reason to make paper applications not stamp hearing order if there's one